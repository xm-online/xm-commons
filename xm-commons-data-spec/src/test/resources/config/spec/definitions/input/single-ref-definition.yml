test-items:
    - key: team/CREATE_EMPLOYEE
      inputSpec: |
          {
            "type": "object",
            "properties": {
              "uaa": {
                "$ref": "#/xmDefinition/create/Employee"
              }
            }
          }

definitions:
    - key: create
      value: |-
          {
            "Employee": {
              "type": "object",
              "properties": {
                "success": {
                  "type": "boolean"
                }
              }
            }
          }
