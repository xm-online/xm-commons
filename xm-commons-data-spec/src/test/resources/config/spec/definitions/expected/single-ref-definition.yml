test-items:
    - key: team/CREATE_EMPLOYEE
      inputSpec: |
          {
            "type": "object",
            "properties": {
              "uaa": {
                "$ref": "#/xmDefinition/create/Employee"
              }
            },
            "xmDefinition": {
              "create": {
                "Employee": {
                  "type": "object",
                  "properties": {
                    "success": {
                      "type": "boolean"
                    }
                  }
                }
              }
            }
          }

definitions:
    - key: create
      value: |-
          {
            "Employee": {
              "type": "object",
              "properties": {
                "success": {
                  "type": "boolean"
                }
              }
            }
          }
